lib/states/account_load.rb

```diff
@@ -5,6 +5,8 @@ def action
         puts I18n.t(:no_active_account_message)
         return @answer = read_input
       end
+      read_login = read_input_with_title(puts(I18n.t(:login_message)))
+      read_password = read_input_with_title(puts(I18n.t(:password_message)))
 
       login = read_login
       password = read_password
```

Comment:
Why not? :)
login = read_input_with_title(puts(I18n.t(:login_message)))
password = read_input_with_title(puts(I18n.t(:password_message)))